src/main/java/org/fogbowcloud/app/jes/arrebol/ArrebolJobSynchronizer.java

```diff
@@ -43,11 +45,34 @@ public JDFJob synchronizeJob(JDFJob job) {
     }
 
     private void updateJob(JDFJob job, ArrebolJob arrebolJob) {
+        updateCommandsState(job.getTaskList(), arrebolJob.getTasks());
         updateTasksState(job.getTaskList(), arrebolJob.getTasks());
         LOGGER.info("Updated tasks state from job [" + job.getId() + "].");
         updateJobState(job, arrebolJob.getJobState());
     }
 
+    private void updateCommandsState(Map<String, Task> iguassuTasks, List<ArrebolTask> arrebolTasks) {
```

Comment:
Please, also update the command exit code, the next feature that we intend to address is the sync of all commands attributes between Iguassu and Arrebol, and doing it now will avoid the need to patch new implementations to Iguassu.